dotnet/src/Extensions/Extensions.StepwisePlanner.UnitTests/Extensions.StepwisePlanner.UnitTests.csproj

```diff
@@ -25,6 +25,6 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Planning.StepwisePlanner\Planning.StepwisePlanner.csproj" />
+    <ProjectReference Include="..\..\SemanticKernel.Abstractions\SemanticKernel.Abstractions.csproj" />
```

Comment:
Is this accurate? Can we merge this into a single Extensions.Planning.UnitTests.csproj?

Reply:
The diff for this file seems misleading. I removed the entire Extensions.StepwisePlanner.UnitTests project because I noticed that Extensions.UnitTests already had a Planning folder with tests for ActionPlanner and SequentialPlanner. So I moved the test file for the SequentialPlanner to that folder.

Reply:
So this file should show up as deleted?

Reply:
Turns out these files were still there until I deleted them outside of VS. I think I deleted through VS originally which merely hid the project within VS which is why these files were still showing up in git.